test/token.test.ts

```diff
@@ -0,0 +1,77 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+// eslint-disable-next-line node/no-missing-import
+import { CliptoToken } from "../typechain";
+import * as constant from "./constant";
+
+describe("CliptoToken", () => {
```

Comment:
In general, test coverage should be better than it is. If we're using upgradable contracts, we should have comprehensive tests for these, especially when it comes to testing an actual upgrade & ensuring that everything works as expected after the upgrade. If we don't have this at a bare minimum, upgradable contracts are going to cause a lot more pain than they assuage.